charts/bpa/templates/_helpers.tpl

```diff
@@ -176,15 +176,26 @@ Return didPrefix
 {{- get (dict "bosch-test" "did:sov:iil:" "idu" "did:sov:idu:") .Values.global.ledger -}}
 {{- end -}}
 
+{{/*
+Return acapy initialization call
+*/}}
+{{- define "acapy.registerLedger" -}}
+{{- if eq .Values.global.ledger "bosch-test" -}}
+curl -d '{\"seed\":\"$(WALLET_SEED)\", \"role\":\"TRUST_ANCHOR\", \"alias\":\"{{ include "bpa.fullname" . }}\"}' -X POST {{ include "bpa.ledgerBrowser" . }}/register;
```

Comment:
Just FYI, this call returns the DID as well
Example
```
{
  "did": "KBvhktVFS6XAMkieYmqqyS",
  "seed": "VkREuZOCy9xIJT02iJxKkHYtrYxY67ka",
  "verkey": "AuyHgcqUrgywzmnfrdMZjYCrDfS4afzRNvFFHJt5aG2x"
}
```
So, you could use the same mechanism to set the correct label on the test network as well.